bundles/admin/admin-layereditor/view/AdminLayerForm/AdditionalTabPane/ServiceMetadata.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Message, Tooltip } from 'oskari-ui';
+import { Controller } from 'oskari-ui/util';
+import { MetadataButton } from './styled';
+
+export const ServiceMetadata = ({ capabilities, controller }) => {
+    const { metadataUuid } = capabilities;
+    if (!metadataUuid) {
+        return (
+            <Fragment>
+                <Message messageKey='metadata.service' />
+                <span>:&nbsp;</span>
+                <Message messageKey='serviceNotAvailable' />
+            </Fragment>
+        );
+    }
+    return (
+        <Tooltip title={metadataUuid}>
+            <Message messageKey='metadata.service' />
+            <MetadataButton onClick={() => controller.showLayerMetadata(metadataUuid)}/>
```

Comment:
We should check if the metadata request is available in the app and not show the button if it's not.